static/js/pivotal.js

```diff
@@ -135,20 +136,31 @@
   $(document).ready(function(){
 
     // add button to story columns
-    var button = '<button class="btn btn-default getStories">get stories</button>';
+    var button = '<button class="btn btn-default getStories">Get stories</button>';
     $(config.selectors.story_column).each(function(){
       $(this).html(button);
     });
 
+
+    // set up listeners on Refresh Button
+    $(config.selectors.refresh_button).click(function(){
+      // display button once more
+      $(this).parents(config.selectors.project).find(config.selectors.story_column).html(button);
+      $(this).parents(config.selectors.project).find('.getStories').click(pivotalButtonOnClickHandler);
+    });
+
     // "get stories" button onClick handler
-    $('.getStories').click(function(e){
-      var $this_button = $(this);
+    $('.getStories').click(pivotalButtonOnClickHandler);
+
+    function pivotalButtonOnClickHandler(e) {
+      var $this_button = $(e.currentTarget);
```

Comment:
@ninotoshi thanks for the heads up. We were following our front-end javascript conventions (i.e., to use camel case conventions for functions, snake cases for variables) as specified in our [internal documents](https://github.com/gengo/docs/blob/master/06_Front-end_Engineer_Conventions/Front-end-Javascript-Conventions.md#general-rules)

Having said that, i will check and fix for inconsistencies in this file :bow: